jenkins/shared-library/vars/githubHelper.groovy

```diff
@@ -13,13 +13,9 @@ boolean isGitHubRepositoryPublic(String orgName, String repoName) {
 }
 
 Object getRepositoryInfo(String token, String ownerName, String repositoryName) {
-	final String getRepoCommand = """
-		curl -L \\
-		  -H "Accept: application/vnd.github+json" \\
-		  -H "Authorization: Bearer ${token}" \\
-		  -H "X-GitHub-Api-Version: 2022-11-28" \\
-		  https://api.github.com/repos/${ownerName}/${repositoryName} \\
-	"""
+	final String getRepoCommand = new StringBuilder().append('curl -L -H "Accept: application/vnd.github+json"')
+		.append(""" -H "Authorization: Bearer ${token}" """)
+		.append("""-H "X-GitHub-Api-Version: 2022-11-28"  https://api.github.com/repos/${ownerName}/${repositoryName}""")
```

Comment:
generally, i find old version more readable due to multiple args per line in new version. consider either multiple appends or something like (excuse the poor formatting):

```groovy
[
		'curl -L',
		  '-H "Accept: application/vnd.github+json"'
		  "-H "Authorization: Bearer ${token}"
		  '-H "X-GitHub-Api-Version: 2022-11-28" '
		 "https://api.github.com/repos/${ownerName}/${repositoryName}"
].join(' ')
```